examples/jsm/helpers/ViewHelper.js

```diff
@@ -54,9 +54,11 @@ class ViewHelper extends Object3D {
 		this.add( zAxis );
 		this.add( yAxis );
 
-		const spriteMaterial1 = getSpriteMaterial( color1 );
-		const spriteMaterial2 = getSpriteMaterial( color2 );
-		const spriteMaterial3 = getSpriteMaterial( color3 );
+		const { posXLabel = null, posYLabel = null, posZLabel = null } = options || {};
```

Comment:
Maybe `labelX`, `labelY` and `labelZ`?  The `pos` prefix somewhat confused me since I thought these properties hold position values.

Reply:
Maybe a different API would also be possible:

```
const viewHelper = new ViewHelper( camera, renderer.domElement );
viewHelper.setLabels( '右', `上`, '前' );
viewHelper.setLabelStyle( 'Bold 24px Arial', '#000000', 20 );
```